meson.build

```diff
@@ -520,6 +520,8 @@ sdb_exe = executable('sdb', ['shlr/sdb/src/main.c'] + sdb_files,
   ],
   implicit_include_directories: false,
   native: true,
+  install: true,
```

Comment:
why this? I'm not sure it is required to make r2 work, this is why it was not installed previously.

Reply:
Its needed if you want radare2-extras to be able to compile sdb texts into sdb binaries.